.gitlab-ci.yml

```diff
@@ -31,13 +31,15 @@ test:unit:
 
 test:govendor-check:
   variables:
-    GOLANG_VERSION: "1.18"
+    GOLANG_VERSION: '1.18'
 
-test:acceptance:
+test:acceptance_tests:
   stage: test
   needs: []
-  except:
-    - /^saas-[a-zA-Z0-9.-]+$/
+  rules:
+    - if: '$CI_COMMIT_TAG =~ /^saas-[a-zA-Z0-9.]+$/'
+      when: never
+    - when: on_success
```

Comment:
why do we need on_success here?

Reply:
to only run if the previous stages are successful (also see [mendertesting](https://github.com/mendersoftware/mendertesting/blob/master/.gitlab-ci-check-docker-acceptance.yml#L72-L75)) - more detailed explanation in the [gitlab ci docs](https://docs.gitlab.com/ee/ci/jobs/job_control.html#rules-examples)

Reply:
okey, but this is pretty much independent, so do we really want to run it on success? is there any repo that does it that way?

Reply:
all repos that use the mendertesting acceptance testing template (so almost all of the backend repos)